src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ErrorProviderTests.cs

```diff
@@ -1259,6 +1259,20 @@ public void ErrorProvider_CustomDataSource_DoesNotThrowInvalidCastException()
             Assert.Null(exception);
         }
 
+        [WinFormsFact]
+        public void ErrorProvider_Icon_NotDisposed_Unexpectedly()
+        {
+            // Unit test for https://github.com/dotnet/winforms/issues/8513.
+            using var provider = new SubErrorProvider();
+            var Icon = provider.Icon;
```

Comment:
This is hard to read with the casing wrong.

```suggestion
            var icon = provider.Icon;
```